.github/workflows/ci.yml

```diff
@@ -38,10 +36,6 @@ jobs:
               os: linux-gcc-14 # This is to use ubuntu 24 and install gcc 14. Should be removed when ubuntu-latest is 26.04
             builder: ubuntu-24.04
             shell: bash
-          - target:
-              os: macos
-              cpu: amd64
-            builder: macos-13
             shell: bash
```

Comment:
I think this should be removed too? otherwise the previous YAML section reads
```yaml
            shell: bash
            shell: bash
```

Reply:
Yeah, definitely. My mistake.